app/views/two_factor_authentication/otp_verification/show.html.erb

```diff
@@ -62,9 +62,8 @@
 <% end %>
 
 <%= render 'two_factor_authentication/troubleshooting_options', presenter: @presenter %>
-
-<% if MfaPolicy.new(current_user).two_factor_enabled? %>
+<% if MfaPolicy.new(current_user).two_factor_enabled? && !in_multi_mfa_selection_flow? %>
   <%= render 'shared/cancel', link: @presenter.cancel_link %>
 <% else %>
-  <%= render 'shared/cancel_or_back_to_options' %>
+  <%= link_to t('two_factor_authentication.choose_another_option'), authentication_methods_setup_path %>
```

Comment:
I don't think the screenshot in the pull request is accurate (anymore), since the partial being removed also rendered the footer with the border, which is no longer present as expected.

![image](https://github.com/18F/identity-idp/assets/1779930/a937d40b-107d-4878-bf60-d66d7cc2460d)